src/persist/src/operators/stream.rs

```diff
@@ -721,6 +694,77 @@ mod tests {
         Ok(())
     }
 
+    /// Test to make sure we handle closing the seal operator correctly and don't
+    /// incorrectly process any seal futures after the operator has been closed.
+    #[test]
+    fn seal_close() -> Result<(), Error> {
+        ore::test::init_logging_default("trace");
```

Comment:
let's not bake trace logging into a test. worst case, if we see a failure somewhere where we wish we'd had trace logging we can cargo stress it with the MZ_LOG_FILTER env var

concretely: either remove this line or make it `ore::test::init_logging()`

Reply:
sorry. this is just following the convention of the other `seal`/`conditional_seal` tests in this file. I'll change this here and elsewhere

Reply:
oh i must have missed those in review. yes please